app/assets/javascripts/teaspoon/jasmine2.coffee

```diff
@@ -1,4 +1,6 @@
 #= require teaspoon/base/teaspoon
+#= require teaspoon/jasmine2/_namespace
```

Comment:
I don't so much like the `_namespace.coffee` file. I think when `jasmine2.coffee` gets loaded, it should first declare it's namespace, then load in all of it's dependencies. The entire contents of this file should at some point be:

```coffeescript
#= require_self
#= require teaspoon/jasmine2/responder
#= require teaspoon/jasmine2/runner
#= require teaspoon/jasmine2/fixture
#= require teaspoon/jasmine2/reporters/console
#= require teaspoon/jasmine2/reporters/html

@Teaspoon.Jasmine2 = {}
```